bindings/py/setup.py

```diff
@@ -92,6 +94,33 @@ def findRequirements(platform):
 
 
 
+class PyTest(TestCommand):
```

Comment:
This is confusing when `pytest` is imported below. I'd call this `TestCommand` (renaming the base class to `BaseTestCommand` when importing it). You could call it `PyTestCommand` but it isn't `python setup.py pytest` so I'd lean towards `TestCommand`.